explorer/models/registryRootNode.ts

```diff
@@ -133,43 +133,43 @@ export class RegistryRootNode extends NodeBase {
             const subs: SubscriptionModels.Subscription[] = this.getFilteredSubscriptions();
 
             const subPool = new AsyncPool(MAX_CONCURRENT_SUBSCRIPTON_REQUESTS);
-            let subsAndRegistries: { 'subscription': SubscriptionModels.Subscription, 'registries': ContainerModels.RegistryListResult, 'client': any }[] = [];
+            let subsAndRegistries: { 'subscription': SubscriptionModels.Subscription, 'registries': ContainerModels.RegistryListResult }[] = [];
             //Acquire each subscription's data simultaneously
             // tslint:disable-next-line:prefer-for-of // Grandfathered in
             for (let i = 0; i < subs.length; i++) {
                 subPool.addTask(async () => {
                     const client = new ContainerRegistryManagement(this.getCredentialByTenantId(subs[i].tenantId), subs[i].subscriptionId);
-                    subsAndRegistries.push({
-                        'subscription': subs[i],
-                        'registries': await client.registries.list(),
-                        'client': client
-                    });
+                    let regs: ContainerModels.Registry[];
+                    try {
+                        regs = await client.registries.list();
+                        subsAndRegistries.push({
+                            'subscription': subs[i],
+                            'registries': regs
+                        });
+                    } catch (error) {
+                        vscode.window.showErrorMessage(error);
```

Comment:
Let caller show error.

Reply:
and other places.

Let me know if the caller is not showing the error.  There are further changes around command registration and error handling that I intend to make.

Reply:
Do you mean handle this within asyncpool? I could do that so its a single location for this.

Reply:
Except change to show parseError(error).message